lib/rx/middleware/response/rx_failed_station.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+module Rx
+  module Middleware
+    module Response
+      class RxFailedStation < Faraday::Response::Middleware
+        include SentryLogging
+
+        def on_complete(env)
+          return unless env.body.is_a? Hash
+
+          station_list = env.body.try(:[], :metadata).try(:[], :failed_station_list)
+          return unless station_list.present?
+
+          message = "Warning: failed station list is not empty, '#{station_list}'"
```

Comment:
Can you add "prescription" somewhere in the message. I know rx will show up in sentry in the stack trace and the request path, but it would make it easier for others to categorize this as an Rx issue if the message said "Warning: prescription failed station list is not empty"